bmq-sdk/src/test/java/com/bloomberg/bmq/it/BrokerSessionIT.java

```diff
@@ -2808,29 +2822,32 @@ public void closeQueueOnPendingConfigureRequest() {
             verifyConfigureRequest(++reqId, server.nextClientRequest());
             verifyCloseRequest(++reqId, server.nextClientRequest(), q1.getParameters());
 
-            // check generated events
+            logger.info("Step 11. Check generated events");
+
             assertEquals(1, events.size());
             verifyBrokerSessionEvent(
                     events.pollLast(), BrokerSessionEvent.Type.e_HOST_HEALTH_RESTORED);
 
-            // Stop the session
+            logger.info("Step 12. Stop the session");
+
             server.pushSuccess();
             assertEquals(GenericResult.SUCCESS, session.stop(TEST_REQUEST_TIMEOUT));
             verifyDisconnectRequest(++reqId, server.nextClientRequest());
 
         } catch (TimeoutException e) {
-            throw new RuntimeException(e);
+            logger.error("Timeout: ", e);
+            fail();
         } catch (Exception e) {
             logger.error("Exception: ", e);
-            throw e;
+            fail();
```

Comment:
I remember last time changing the catch block to catch Throwable (which include both Exception and AssertionError) helped me to find the error